src/test/java/com/thebois/models/beings/ColonyTests.java

```diff
@@ -0,0 +1,55 @@
+package com.thebois.models.beings;
+
+import org.junit.jupiter.api.Test;
+
+import com.thebois.models.Position;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class ColonyTests {
+
+    @Test
+    public void createBeingEqualTo3Test() {
+
+        // arrange
+        final Colony colony = new Colony();
+        // Act
+        colony.createBeing();
+        colony.createBeing();
+        colony.createBeing();
+        // Assert
+        assertThat(colony.getBeings().size()).isEqualTo(3);
+        assertThat(colony.getBeings().size()).isNotEqualTo(4);
```

Comment:
Per definition it can not be equal to 4 if it is equal to 3.

Reply:
I was only going to keep one of them! Thanks for spotting.